simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountFragment.kt

```diff
@@ -894,29 +841,22 @@ class AccountFragment : Fragment() {
           view,
           object : com.squareup.picasso.Callback {
             override fun onSuccess() {
-              this@AccountFragment.uiThread.runOnUIThread {
-                onSuccess.invoke()
-              }
+              onSuccess.invoke()
```

Comment:
Are the Picasso callbacks guaranteed to be called on the UI thread? It's been a very long time, I'm unsure what guarantees they provide.

Reply:
I couldn't find any guarantee in the doc. Looking at the code, it is so though. Everything happens on the same thread on completion: the view image is set and then the callback is called. I think it's unlikely to change in the future.

Reply:
Yeah, I doubt they would change this.